app/src/pages/inside/common/statusDropdown/statusDropdown.jsx

```diff
@@ -46,34 +49,41 @@ const messages = defineMessages({
 @connect(
   (state) => ({
     currentProject: activeProjectSelector(state),
+    currentPage: pageSelector(state),
```

Comment:
Not need to get the page type here, you can specify the necessary prop from the place where this component used.